source/components/grid/index.html

```diff
@@ -415,232 +307,283 @@ <h3 class="hxSubSectionTitle" id="responsive">Responsive Layout</h3>
-          </p>
+        <div class="hxCol hxSpan-12">12
+          <div class="hxRow">
+            <div class="hxCol hxSpan-6">6
+              <div class="hxRow">
+                <div class="hxCol hxSpan-4">4</div>
+                <div class="hxCol hxSpan-4">4</div>
+                <div class="hxCol hxSpan-4">4</div>
+              </div>
+            </div>
+            <div class="hxCol hxSpan-6">6
+              <div class="hxRow">
+                <div class="hxCol hxSpan-3">3</div>
+                <div class="hxCol hxSpan-9">9</div>
+              </div>
+            </div>
+          </div>
         </div>
       </div>
     </div>
   </div>
   {% code 'html' %}
     <div class="hxRow">
-      <div class="hxCol-sm-4 hxCol-md-3 hxCol-lg-2">...</div>
-      <div class="hxCol-sm-4 hxCol-md-3 hxCol-lg-2">...</div>
-      <div class="hxCol-sm-4 hxCol-md-3 hxCol-lg-2">...</div>
-      <div class="hxCol-sm-4 hxCol-md-3 hxCol-lg-2">...</div>
-      <div class="hxCol-sm-4 hxCol-md-3 hxCol-lg-2">...</div>
-      <div class="hxCol-sm-4 hxCol-md-3 hxCol-lg-2">...</div>
-      <div class="hxCol-sm-4 hxCol-md-3 hxCol-lg-2">...</div>
+      <div class="hxCol hxSpan-12">12
+        <div class="hxRow">
+          <div class="hxCol hxSpan-6">6
+            <div class="hxRow">
+              <div class="hxCol hxSpan-4">4</div>
+              <div class="hxCol hxSpan-4">4</div>
+              <div class="hxCol hxSpan-4">4</div>
+            </div>
+          </div>
+          <div class="hxCol hxSpan-6">6
+            <div class="hxRow">
+              <div class="hxCol hxSpan-3">3</div>
+              <div class="hxCol hxSpan-9">9</div>
+            </div>
+          </div>
+        </div>
+      </div>
     </div>
   {% endcode %}
 </section>
 
-<section>
-  <h3 class="hxSubSectionTitle" id="styling">Styling the Grid</h3>
-  <div class="hxAlert">
+<section><!-- styling containers -->
+  <h2 class="hxSectionTitle" id="styling-containers">Styling Containers</h2>
+  <div class="hxAlert hxAlert--warning">
     <div class="hxAlert__icon">
-      <hx-icon type="info-circle"></hx-icon>
+      <hx-icon type="exclamation-triangle"></hx-icon>
     </div>
     <div class="hxAlert__text">
-      <span class="hxAlert__status">NOTE</span>
-      Avoid styling rows and columns.<br />
-      Add a container within a column to apply styling.
+      <span class="hxAlert__status">Warning</span>
+      Do not attempt to style Rows or Columns directly.
     </div>
   </div>
   <br />
-
-  <p>
-    Because styles for rows and columns are meticulously defined to support
-    the grid definition, any modification to the <code>margin</code>,
-    <code>padding</code>, or geometry of these elements will affect the
-    layout.
-  </p>
-
   <ul>
-    <li>
-      Wrap your content in a <code>&lt;div&gt;</code> and style
-      using a custom CSS class.
-    </li>
+    <li>Rows have margins set to align the outer Columsn with surrounding content.</li>
```

Comment:
s/`Columsn`/`Columns` (also, why are `Columns` and `Rows` capitalized globally?

Reply:
They are capitalized to promote distinction as a type of container, rather than the more generic concept of a row/column.

Reply:
Now that you mention this, I'm going to go ahead and modify the capitalization.